src/analyses/constant_propagator.cpp

```diff
@@ -20,6 +20,8 @@ Author: Peter Schrammel
 #include <util/simplify_expr.h>
 #include <util/cprover_prefix.h>
 
+#include <langapi/language_util.h>
+
```

Comment:
What does this change have to do with the subject of the PR?

Reply:
It was a missing header, which was hidden by the fact that goto_program_template.h included it.